build.gradle.kts

```diff
@@ -80,14 +81,39 @@ dependencies {
 	include(implementation("com.oroarmor", "aftermath", "1.0.0-beta"))
 
 	include("org.joml", "joml", "1.10.1")
+	include("org.lz4", "lz4-java", "1.8.0")
+	api(platform("org.lwjgl:lwjgl-bom:$lwjglVersion"))
+
+	include("org.lwjgl", "lwjgl", lwjglVersion)
+	include("org.lwjgl", "lwjgl-jemalloc", lwjglVersion)
+	include("org.lwjgl", "lwjgl-openal", lwjglVersion)
+	include("org.lwjgl", "lwjgl-stb", lwjglVersion)
+	include("org.lwjgl", "lwjgl-tinyfd", lwjglVersion)
+	include("org.lwjgl", "lwjgl-assimp", lwjglVersion)
+	include("org.lwjgl", "lwjgl-glfw", lwjglVersion)
 	include("org.lwjgl", "lwjgl-shaderc", lwjglVersion)
+	include("org.lwjgl", "lwjgl-stb", lwjglVersion)
 	include("org.lwjgl", "lwjgl-vma", lwjglVersion)
 	include("org.lwjgl", "lwjgl-vulkan", lwjglVersion)
-	include("org.lwjgl", "lwjgl-shaderc", lwjglVersion, classifier = lwjglNatives)
-	include("org.lwjgl", "lwjgl-vma", lwjglVersion, classifier = lwjglNatives)
+	include("org.lwjgl", "lwjgl-vma", lwjglVersion)
+	include("org.lwjgl", "lwjgl-jemalloc", lwjglVersion)
+	include(runtimeOnly("org.lwjgl", "lwjgl-shaderc", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-vma", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-jemalloc", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-openal", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-stb", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-tinyfd", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-assimp", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-glfw", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-stb", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-vma", classifier = lwjglNatives))
+	include(runtimeOnly("org.lwjgl", "lwjgl-jemalloc", classifier = lwjglNatives))
+
+	include("it.unimi.dsi", "fastutil-core", "8.5.4")
 
 	if (lwjglNatives == "natives-macos" || lwjglNatives == "natives-macos-arm64") {
-		include("org.lwjgl", "lwjgl-vulkan", lwjglVersion, classifier = lwjglNatives)
+		include(runtimeOnly("org.lwjgl", "lwjgl-vulkan", classifier = lwjglNatives))
```

Comment:
have you tested this on mac's?

Reply:
It will need to always be included. But only added as a dev dependency on mac

Reply:
It works when I manually replace `OperatingSystem.current()` with `OperatingSystem.MAC_OS`
And it somehow works without that library on windows on production environment without that native library.

Reply:
No have the runtime only be when the operation system is Mac, but always include it

Reply:
Also we probably should use shadow and repackage this to not conflict with minecrafts lwjgl

Reply:
Java refuses to load 2 native libraries with the same name at the same time from my testing so I don't expect repackaging will work.

Reply:
it should be fine because their natives have a different name i believe